olsrd/files/olsrd.sh

```diff
@@ -1,10 +1,5 @@
 #!/bin/sh /etc/rc.common
-# Copyright (C) 2008-2017 OpenWrt.org
-
-START=65
-
-SERVICE_DAEMONIZE=1
-SERVICE_WRITE_PID=1
+# Copyright (C) 2008-2021 OpenWrt.org
```

Comment:
Please leave the original copyright statements alone. Either add your own (Copyright (C) 2021 Nick Hainke <vincent@systemli.org>) or don't do anything about it. The same applies for other instances of the OpenWrt copyright.

Reply:
Okay, thanks for clarifying.

Reply:
changing to spdx is okay?